beluga/cmake/Config.cmake.in

```diff
@@ -1,11 +1,15 @@
 @PACKAGE_INIT@
 
+list(APPEND CMAKE_MODULE_PATH "/usr/lib/x86_64-linux-gnu/cmake/OpenVDB/")
+list(APPEND CMAKE_MODULE_PATH "/usr/lib/aarch64-linux-gnu/cmake/OpenVDB")
+
 include(CMakeFindDependencyMacro)
 find_dependency(Eigen3 REQUIRED NO_MODULE)
 find_dependency(range-v3 REQUIRED)
 find_dependency(HDF5 COMPONENTS CXX REQUIRED)
 find_dependency(Sophus REQUIRED)
 find_dependency(TBB REQUIRED)
+find_dependency(OpenVDB REQUIRED)
```

Comment:
@pvela2017 this should not be required. It may be missing.

Reply:
Fixed in 88966dc297dd2e60b9d5d3b015e02b400a5305ea